frappe/templates/print_formats/standard_macros.html

```diff
@@ -135,7 +135,7 @@
 	{% elif df.fieldtype=="Check" and doc[df.fieldname] %}
 		<!-- <i class="{{ 'fa fa-check' }}"></i> -->
 		<svg viewBox="0 0 16 16"
-			fill="transparent" stroke="var(--icon-stroke)" stroke-width="2"
+			fill="transparent" stroke="#1F272E" stroke-width="2"
```

Comment:
Can we use a scss variable here ?

Reply:
We were using the scss variable only but it was not working for PDF generation.